tests/e2e/actor_reminder/actor_reminder_test.go

```diff
@@ -243,3 +244,47 @@ func TestActorReminderPeriod(t *testing.T) {
 		require.Equal(t, 5, count)
 	})
 }
+
+func TestActorReminderTTL(t *testing.T) {
+	externalURL := tr.Platform.AcquireAppExternalURL(appName)
+	require.NotEmpty(t, externalURL, "external URL must not be empty!")
+
+	logsURL := fmt.Sprintf(actorlogsURLFormat, externalURL)
+
+	// This initial probe makes the test wait a little bit longer when needed,
+	// making this test less flaky due to delays in the deployment.
+	t.Logf("Checking if app is healthy ...")
+	_, err := utils.HTTPGetNTimes(externalURL, numHealthChecks)
+	require.NoError(t, err)
+
+	// Set reminder
+	reminder := actorReminder{
+		Data:    "reminderdata",
+		DueTime: "1s",
+		Period:  "PT2S",
+		TTL:     "9s",
+	}
+	reminderBody, err := json.Marshal(reminder)
+	require.NoError(t, err)
+
+	t.Run("Actor reminder with TTL should run correct number of times", func(t *testing.T) {
+		reminderName := "ttl-reminder"
+		actorID := "ttl-reminder-actor"
+		_, err = utils.HTTPDelete(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName))
+		require.NoError(t, err)
+		// Registering reminder
+		_, err = utils.HTTPPost(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName), reminderBody)
+		require.NoError(t, err)
+
+		t.Logf("Sleeping for %d seconds ...", secondsToCheckReminderResult)
+		time.Sleep(secondsToCheckReminderResult * time.Second)
+
+		t.Logf("Getting logs from %s to see if reminders did trigger ...", logsURL)
+		resp, err := utils.HTTPGet(logsURL)
+		require.NoError(t, err)
+
+		t.Log("Checking if all reminders did trigger ...")
+		count := countActorAction(resp, actorID, reminderName)
+		require.Equal(t, 5, count)
```

Comment:
Does this have to be exact match or can there be legit variation?

Reply:
I left a margin of 2 seconds, so I expect the exact match